pkg/reconciler/pubsub/pullsubscription/keda/controller.go

```diff
@@ -109,6 +114,11 @@ func NewController(
 		Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 	})
 
+	serviceAccountInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
+		FilterFunc: controller.FilterGroupVersionKind(duckv1alpha1.SchemeGroupVersion.WithKind("pullsubscription")),
```

Comment:
should be PullSubscription, and also from pubsub APIs not the duckv1alpha1... it's from the pkg/apis/pubsub/v1alpha1